metadeploy/labels_es-MX.json

```diff
@@ -9,7 +9,7 @@
             "description": "tagline of product"
         },
         "description": {
-            "message": "¡Le damos la bienvenida al Instalador del Módulo de gestión de programas (PMM)!\r\n\r\nEl Módulo de gestión de programas es un marco de trabajo estándar para el seguimiento, la gestión y la entrega de programas, servicios y entregas de servicio.\r\n\r\n*IMPORTANTE*: Salesforce no recomienda instalar actualizaciones directamente en organizaciones de producción. En su lugar, instale el Módulo de gestión de programas en uno de sus entornos sandbox u organizaciones de desarrollador gratuitos primero.\r\n\r\n*Espacio para párrafo legal si fuese necesario*\r\n\r\nUna vez instalado el Módulo de gestión de programas, consulte los recursos en nuestra documentación.\r\n\r\n* <Documentación del Módulo de gestión de programas>\r\n\r\n¿Preguntas? Conviértase en miembro del grupo de clientes <Módulo de gestión de programas> en Power of Us Hub.",
+            "message": "¡Le damos la bienvenida al Instalador del Módulo de gestión de programas (PMM)!\r\n\r\nEl Módulo de gestión de programas es un marco de trabajo estándar para el seguimiento, la gestión y la entrega de programas, servicios y entregas de servicio.\r\n\r\n*IMPORTANTE*: Salesforce no recomienda instalar actualizaciones directamente en organizaciones de producción. En su lugar, instale el Módulo de gestión de programas en uno de sus entornos sandbox u organizaciones de desarrollador gratuitos primero.\r\n\r\nSalesforce.org Program Management Module (“PMM”) is an extensible application with open source and closed source components to address the unique core needs of program management. PMM utilizes a Salesforce.com, Inc. (“Salesforce”) managed package to provide nonprofits with a path to centralize programmatic information in order to more effectively manage and deliver programs and services. Program Management Module provides out-of-the-box translations for Spanish, French, German, Dutch, and Japanese.\r\n\r\nUna vez instalado el Módulo de gestión de programas, consulte los recursos en nuestra documentación.\r\n\r\n* [Documentación del Módulo de gestión de programas](https://powerofus.force.com/s/article/PMM-Documentation)\r\n\r\n¿Preguntas? Conviértase en miembro del grupo de clientes [Módulo de gestión de programas](https://powerofus.force.com/s/group/0F91E000000bs8XSAQ/program-management-module) en Power of Us Hub.",
```

Comment:
Are we intentionally adding English into the translated paragraph?

Reply:
Yes. The translated version is not yet available from SFDO Legal. (@meggraySFDO)